### ISSUE TITLE
Player visibility configuration doesnt work

### ISSUE THREAD

**Issue by elizeuangelo** [unsoluble/smalltime#46]
Im using 0.8.6 with DnD5e 1.3.3, and when I change the visibility settings it seems to have no effect, I cant disable nor set on `sun/moon position only` mode.

**Comment by unsoluble** [comment]
Are you making this change in the scene configuration screen, or only in the module settings window? (which only sets the default for new scenes going forward)

**Comment by unsoluble** [comment]
Thanks for the report, though! You're the second person to make that assumption, so it's clear I could do some work on the wording in there. I'm curious, do you run Foundry in English, or one of the translations?

**Comment by elizeuangelo** [comment]
In English. I believe the scene controls could have a 'default' option wich gets the controls from the settings itself, so you dont have to change every single scene in case you want to change the standard behavior.

**Comment by unsoluble** [comment]
Yeah that's a pretty good idea, actually. Lemme think that through, see how it goes. :)

**Comment by unsoluble** [comment]
Made this change for 0.7.2.
